drivers/chown_unix.go

```diff
@@ -101,7 +101,8 @@ func (c *platformChowner) LChown(path string, info os.FileInfo, toHost, toContai
 	}
 	if uid != int(st.Uid) || gid != int(st.Gid) {
 		cap, err := system.Lgetxattr(path, "security.capability")
-		if err != nil && !errors.Is(err, system.EOPNOTSUPP) && !errors.Is(err, system.EOVERFLOW) && err != system.ErrNotSupportedPlatform {
+		if err != nil && !errors.Is(err, system.EOPNOTSUPP) && !errors.Is(err, syscall.ENOTSUP) &&
```

Comment:
The direct reference to `syscall.E…` here breaks the abstraction to an extent; could that error be published by the `system` subpackage as well?

(Alternatively, do we really need all three of `system.ENOTSUP`, `system.EOPNOTSUPP`, `system.ErrNotSupportedPlatform`? I know very little about extended attribute implementations/compatibility.)

Reply:
I inspected `lgetxattr(2)`, `lsetxattr(2)`, and `llistxattr(2)` on Linux, as well as `getxattr(2)`, `setxattr(2)`, and `listxattr(2)` on MacOS. I found that these system calls actually return `ENOTSUP` instead of `EOPNOTSUPP`. Hence, `system.ENOTSUP` and `system.EOPNOTSUPP` are sufficient.

Note that `EOPNOTSUPP` and `ENOTSUP` have the same value in Linux according to `errno(3)`.

Reply:
For the record, note to self:
- The UNIX spec says `EOPNOTSUPP` is for sockets, `ENOTSUP` for other situations (and allows the two to be the same value).
- macOS apparently differentiates the two in the kernel, but maps them back to the same value in libc in most modes
- Linux kernel uses `EOPNOTSUPP` exclusively (and in the extended attribute code), glibc defines `ENOTSUP` to be equal.